### DISCUSSION TITLE
shell

### DISCUSSION THREAD

**Discussion by lildevil109** [community/community#62331] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I installed your update and the start rectangle moved to the search rectangle at the bottom of my page. Instead of saying start all it has is your shell logo. To not know how how to undo the update and reinstall. Using windows11.

Dan

**Comment by diyaryvzz** [comment]
Can you explain a little more why I should not use Windows 11.
